h/static/styles/help-page.scss

```diff
@@ -152,6 +152,7 @@ a {
   list-style-type: none;
 
   &:before {
+    box-sizing: content-box;
```

Comment:
What was the problem here?

Reply:
This is actually a fix for a bug that was introduced with the Normalize.css bump. The red circles around the numbers in the numbered list at the top of the page apparently depend on the `content-box` model, because the global `border-box` declaration in Normalize collapsed their height a bit, making them look squished.